packages/okta-react/src/SecureRoute.js

```diff
@@ -78,6 +77,15 @@ export default withAuth(class SecureRoute extends Component {
   }
 
   render() {
-    return <Route path={this.props.path} render={this.createRenderWrapper} />;
+    return (
+      <Route
+        exact={this.props.exact}
+        path={this.props.path}
+        render={this.createRenderWrapper}
+      />
+    );
   }
-});
+}
+
+export { SecureRoute };
```

Comment:
this is the unwrapped version of SecureRoute. This makes it easier to test, because now we can mock the props that withAuth was passing before.